parquet/src/file/reader.rs

```diff
@@ -45,23 +45,39 @@ pub trait Length {
     fn len(&self) -> u64;
 }
 
-/// The ChunkReader trait generates readers of chunks of a source.
+/// Generates [`Read`]ers to read chunks of a Parquet data source.
 ///
-/// For more information see [`File::try_clone`]
+/// The Parquet reader uses [`ChunkReader`] to access Parquet data, potentially
+/// decoding in parallel from from the same data source. The returned readers
```

Comment:
```suggestion
/// decoding concurrently from from the same data source. The returned readers
```

If one wants to be pedantic, this can't be used in parallel as it has no synchronisation